### DISCUSSION TITLE
Cannot Save Project File

### DISCUSSION THREAD

**Discussion by thenags1** [mifi/lossless-cut#1780] [Support]
This used to work fine but for the last 2 versions every time I'm working on something, no matter the location I have Project Files set to save to, I get the error 'Cannot Save Project File'.  Any ideas how to fix this?

**Comment by mifi** [comment]
I need some more info, maybe you can try to look at this: https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&projects=&template=bug.yml
